### PR TITLE
"Beta" version

### PR DESCRIPTION
**Pull request by reoliveira**
All implemented system components are integrated as of this pull request. Data is collected and sorted  by autonomous threads and information travels full route through the system.

### CODE REVIEW

**Comment by edolinsky** [comment]
Resolves #22 #11 #10

File: app/controllers/ContentController.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        Logger.info("ContentController: Starting application at " + DateTime.now().toString());
+        sortNotification = new ThreadNotification();
+
+
+        sorter = new Thread(new SortingNode(access, sortNotification));
+        sorter.start();
+
+        collectors = new ArrayList<>();
+        Thread twitter = new Thread(new DataCollectionRunner(new JavaDataCollector(access, new
+                TwitterSource()), sortNotification)); // TODO THIS IS AWFUL
```
Comment (edolinsky) [inline app/controllers/ContentController.java:35]: If I'm not mistaken, this is still awful. May be slightly better (at the very least easier to read) to make local instances and pass them in?

File: conf/routes [hunk 2]
```diff
@@ -4,10 +4,6 @@
 
 # An example controller showing a sample home page
 GET     /                           controllers.HomeController.index
-# An example controller showing how to use dependency injection
-GET     /count                      controllers.CountController.count
-# An example controller showing how to write asynchronous code
-GET     /message                    controllers.AsyncController.message
```
Comment (edolinsky) [inline conf/routes:10]: +1 for all this cleanup

File: build.sbt [hunk 3]
```diff
@@ -19,5 +19,4 @@ libraryDependencies ++= Seq(
 )
 
 
-fork in run := true
```
Comment (sammiej) [inline build.sbt:22]: This line kept me from being able to start my server, do we need to keep it?

Comment (edolinsky) [reply build.sbt:22]: Nope! Probably why it's removed (in red).

File: app/controllers/ContentController.java (hunk 1 above)
Comment (edolinsky) [reply app/controllers/ContentController.java:35]: Should be fine for now; as long as we address the `todo` later.

File: build.sbt (hunk 3 above)
Comment (sammiej) [reply build.sbt:22]: Oh, I meant that I removed it so that it would start server properly, so I wasn't sure if that was the right choice.

**Review by edolinsky — APPROVED** [review]
Looks good! Pulling in so we can keep progressing.

File: build.sbt (hunk 3 above)
Comment (edolinsky) [reply build.sbt:22]: Ah, that makes much more sense. Might need to change it back later, but I'm ok with leaving it out for now. @reoliveira @MariaStrasky, thoughts?

Comment (edolinsky) [reply build.sbt:22]: Based on [this](http://www.scala-sbt.org/0.12.3/docs/Detailed-Topics/Forking.html), we should be ok.

**Pull request merged**